### PR TITLE
RMG: Capitalize section headings (=head3)

### PR DESCRIPTION
**Pull request by rwp0**
Capitalize `=head3`'s for consistency

Dual-Life is being a special term:

https://github.com/Dual-Life

### CODE REVIEW

File: Porting/release_managers_guide.pod [hunk 1]
```diff
@@ -400,7 +400,7 @@ colon-delimited versions to use for comparison.  For example:
 
 L<http://analysis.cpantesters.org/beforemaintrelease?pair=5.20.2:5.22.0%20RC1>
 
-=head3 check pod errors
+=head3 cCeck pod errors
```
Comment (rwp0) [inline Porting/release_managers_guide.pod]: Working on this

File: Porting/release_managers_guide.pod [hunk 2]
```diff
@@ -1278,27 +1278,27 @@ branch to avoid the merge commit (as you might normally do when merging a
 small branch into blead) since doing so will invalidate the tag that you
 created earlier.
 
-=head3 publish the release tag
+=head3 Oublish the release tag
```
Comment (rwp0) [inline Porting/release_managers_guide.pod]: P

**Review by rwp0 — COMMENTED** [review]
Spotted 2 spelling errors, will amend

**Comment by rwp0** [comment]
#### Testing the individual file locally:

```
./perl -Ilib ./t/porting/podcheck.t
ok 297 - POD of Porting/release_managers_guide.pod, excluding 1 not shown known potential problems
```

#### Failed GH action test
https://github.com/rwp0/perl5/actions/runs/5918255806/job/16047390772#step:12:96:
```
Test Summary Report
-------------------
porting/podcheck.t      (Wstat: 0 Tests: 1468 Failed: 1)
  Failed test:  297
Files=34, Tests=45284, 21 wallclock secs ( 1.75 usr  0.12 sys + 36.13 cusr  1.63 csys = 39.63 CPU)
Result: FAIL
Finished test run at Sun Aug 20 15:49:54 2023.
Error: Process completed with exit code 1.
```

**Comment by rwp0** [comment]
Amended reading the below error output.
Capitalized the `L<>` references as well (it seems to be case-sensitive).
Hoping the checks will pass now.

```
porting/readme.t .......... ok
#   "unresolved internal link"  (6 occurrences, expected 0)
#     'update Module::CoreList' near line 440 of Porting/release_managers_guide.pod
#     'update INSTALL' near line 467 of Porting/release_managers_guide.pod
#     'update Module::CoreList' near line 511 of Porting/release_managers_guide.pod
#     'build the tarball' near line 595 of Porting/release_managers_guide.pod
#     'finalize perldelta' near line 7[36](https://github.com/Perl/perl5/actions/runs/5918256056/job/16047475225?pr=21404#step:12:37) of Porting/release_managers_guide.pod
#     'update INSTALL' near line 1[39](https://github.com/Perl/perl5/actions/runs/5918256056/job/16047475225?pr=21404#step:12:40)5 of Porting/release_managers_guide.pod
# See end of this test output for your options on silencing this
# 
# HOW TO GET porting/podcheck.t TO PASS
# 
# There was 1 file that had new potential problems identified.
# Some of them may be real, and some of them may be false positives because
# this program isn't as smart as it likes to think it is.  You can teach this
# program to ignore the issues it has identified, and hence pass, by doing the
# following:
# 
# 1) If a problem is about a link to an unknown module or man page that
#    you know exists, re-run the command something like:
#       ./perl -I../lib porting/podcheck.t --add-link { MODULE | man_page ... }
#    (MODULEs should look like Foo::Bar, and man_pages should look like
#    bar(3c); don't do this for a module or man page that you aren't sure
#    about; instead treat as another type of issue and follow the
#    instructions below.)
# 
# 2) For other issues, decide if each should be fixed now or not.  Fix the
#    ones you decided to, and rerun this test to verify that the fixes
#    worked.
# 
# 3) If there remain false positive or problems that you don't plan to fix right
#    now,
#    run this test script by hand, using the following formula (on
#    Un*x-like machines):
#         cd t
#         ./perl -I../lib porting/podcheck.t --regen
# 
#    That should cause all current potential problems to be accepted by
#    the program, so that the next time it runs, they won't be flagged.
#    For a few files, such as perltoc, certain issues will always be
#    expected, and more of the same will be added over time.  For those,
#    before you do the regen, you can edit
#    /home/runner/work/perl5/perl5/t/porting/known_pod_issues.dat
#    and find the entry for the module's file and specific error message,
#    and change the count of known potential problems to -1.
porting/podcheck.t ........ 
Failed 1/1[46](https://github.com/Perl/perl5/actions/runs/5918256056/job/16047475225?pr=21404#step:12:47)8 subtests 
	(less 10[75](https://github.com/Perl/perl5/actions/runs/5918256056/job/16047475225?pr=21404#step:12:76) skipped subtests: 392 okay)
```

https://github.com/Perl/perl5/actions/runs/5918256056/job/16047391065?pr=21404

File: Porting/release_managers_guide.pod [hunk 3]
```diff
@@ -197,7 +197,7 @@ The work of building a release candidate for an even numbered release
 release candidate.  Some of the following steps should be done regularly,
 but all I<must> be done in the run up to a release.
 
-=head3 dual-life CPAN module synchronisation
+=head3 Dual-Life CPAN module synchronisation
```
Comment (tonycoz) [inline Porting/release_managers_guide.pod:200]: I don't think "Life" needs to be capitalized here.

In this context the document isn't referring only to modules in the github Dual-Life organization, so applying the capitalization from the organization doesn't make sense.

Comment (rwp0) [reply Porting/release_managers_guide.pod:200]: Can a maintainer amend my commit for this? I tried, and I'm afraid to mess up git history by pushing 🙂

**Comment by rwp0** [comment]
See https://github.com/Perl/perl5/pull/21463

File: Porting/release_managers_guide.pod (hunk 3 above)
Comment (rwp0) [reply Porting/release_managers_guide.pod:200]: Resolved in https://github.com/Perl/perl5/pull/21463

**Pull request closed without merging**